src/event_emitter.ts

```diff
@@ -204,12 +191,8 @@ export class EventEmitter {
    * @description
    *     Send a message to a channel, excluding the sender.
    *
```

Comment:
`@description` is left in

Reply:
fixed: https://github.com/drashland/sockets/pull/51/commits/3bad69b22ceb5f70569bf53c73cfc63738ea3028